master.js

```diff
@@ -0,0 +1,151 @@
+let data = [];
+let query = (document.querySelector("input:first-child:last-child").value).toLowerCase()
+let isLoading = true;
+
+async function fetchData (){
+    const response = await fetch("https://dummyjson.com/todos")
+    const {todos} = await response.json()
+    data = todos;
+    if(window.localStorage.getItem("list"))
+        data = JSON.parse(window.localStorage.getItem("list"));
+}
+
+
+fetchData().then(() => {
+    isLoading = false;
+    fillTable(data)
+})
+
+function fillTable(givenData){
+    let loader = document.querySelector(".loading");
```

Comment:
can we create a new file called for example **helpers.js** and add functions to get the DOM elements, 

e.g 
function getLoader(){
return document.querySelector(".loading")
}

and the same for the rest of the DOM elements